feather/server/src/entities.rs

```diff
@@ -57,3 +58,8 @@ fn spawn_living_entity(entity: &EntityRef, client: &Client) -> SysResult {
     client.send_living_entity(network_id, uuid, pos, kind);
     Ok(())
 }
+
+/// Increasing distance of an entity falling. Used to calculate
+/// fall damage for entities with health.
```

Comment:
Assuming I understand what this means, I think it would be best to say something similar to 
`/// Distance fallen since an entity was on solid ground.` 
Without the explanation of what the type is for.